poetry/masonry/builders/builder.py

```diff
@@ -146,8 +146,9 @@ def find_files_to_add(self, exclude_build=True):  # type: (bool) -> list
         )
         to_add.append(Path("pyproject.toml"))
 
-        # If a license file exists, add it
-        for license_file in self._path.glob("LICENSE*"):
+        # If any license files exist, add them.
+        for license_filename in list(self._package.license_files):
```

Comment:
Is it necessary to wrap this in a `list` call here?

Reply:
Just checked and no it looks like I don't. I guess I forgot that sets are itterable when I was writing that bit. Fix incoming.